sources/academy/platform/get_most_of_actors/store_basics/how_actor_monetization_works.md

```diff
@@ -33,18 +33,51 @@ Monetizing your Actor on the Apify platform involves several key steps:
   - 2nd user starts their trial but pays next month.
   - 3rd user on a free plan finishes the trial without upgrading to a paid plan and can’t use the Actor further.
 
+Learn more about rental pricing model in our [documentation](/platform/actors/publishing/monetize#rental-pricing-model).
+
 ### Pay-per-result pricing model
 
 ![pay per result model example](images/ppr-model.png)
 
-- _How it works_: you charge users based on the number of results your Actor generates. You earn 80% of the revenue minus platform costs.
+- _How it works_: you charge users based on the number of results your Actor generates. You earn 80% of the revenue minus platform usage costs.
+- _Profit calculation_: `profit = (0.8 * revenue) - platform usage costs`
 - _Cost breakdown_:
   - Compute unit: $0.4 per CU
   - Residential proxies: $13 per GB
   - SERPs proxy: $3 per 1,000 SERPs
   - Data transfer (external): $0.20 per GB
   - Dataset storage: $1 per 1,000 GB-hours
-- _Example_: you set a $1 per 1,000 results price. Two users generate 50,000 and 20,000 results, paying $50 and $20, respectively. If platform costs are $5 and $2, your profit is $49.
+- _Example_: you set a $1 per 1,000 results price. Two users generate 50,000 and 20,000 results, paying $50 and $20, respectively. If platform usage costs are $5 and $2, your profit is $49.
```

Comment:
_Example_: you set a price of $1 per 1,000 results. Two users generate 50,000 and 20,000 results, paying $50 and $20, respectively. If the platform usage costs are $5 and $2, your profit is $49.